script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_3.txt

```diff
@@ -93,52 +93,52 @@ For some reason, Senpai smiles happily.
 "?"
 Arihiko and I look at each other and tilt our heads.
 <Page29>
-"I envy you. I admire friends who can understand each other like that without any hard feelings."
+"I really envy you. I wish I had friends who can understand eachother without all the fervor."//「羨ましいなあ。そうやって何の気負いもなしで分かり合える友人がいるのって、憧れます」"really" added to make it more casual
 Senpai breathes out in approval.
 ""Is that so?""
 Arihiko and I look at each other, furrowing our brows.
 <Page30>
-"Well, it seems it's about time. I'm going back, but Tohno-kun, did you see the news this morning?"
-"Oh, ...I've been so busy I forgot. I haven't seen it yet."
+"Well, it seems it's about that time. I'm going back, but before I do, Tohno-kun, did you see the news this morning?"//「と、そろそろ時間ですね。わたしは戻りますけど、遠野くん、今朝のニュース見ましたか？」
+"Oh... I've been so busy that I forgot. I haven't seen it yet."
 <Page31>
 "I see. Well then, let me ask you directly. There was a large mansion on the news this morning. Was that your new house?"
 "Eh?"
 <Page32>
 Now that she mentions it, I remember Hisui saying that the police came by to question us.
 <Page33>
-"Ah, that must have been talking about my home.
+"Ah, yeah, that was definitely my house.//「ああ、それはきっと<家|うち>のコトかもですね。
 I heard that the police came by to talk to us this morning."
-"I see. Tohno-kun, you shouldn't play around too much at night."
+"I see. Tohno-kun, you shouldn't play around too much at night."//「そうですか。遠野くん、あんまり夜遊びしちゃダメですよ」 夜遊び is "nightlife" but I don't actually have a better translation than this on hand.
 <Page34>
 Senpai quickly walks away.
 We watch her depart in silence.
 <Page35>
 "Mm-hmm ―――Now then, hey Tohno."
 "What is it? I won't answer you if it's something stupid."
-"It's not stupid. The big question is, when did you become so close to Senpai that she started coming to see you?"
+"It's not stupid. The million-dollar question is, when did you become so close with Senpai that she has started coming to see you?"//「つまらなくはないぞ。大きな疑問として、オマエはいつのまに、先輩の方から会いに来るほど親しくなってたんだ？」
```

Comment:
Not sure the `has` in "she has started coming" belongs here? Feels off when read with the rest of the sentence